README.md

```diff
@@ -113,12 +113,12 @@ To run integration tests
 ```
 ## Publishing Changes
 
-In order to publish changes, you will need to fork the repository or ask to join the `ibm-developer` org and branch off the `development` branch.
+In order to publish changes, you will need to fork the repository or ask to join the `ibm-developer` org and branch off the `master` branch.
 
-Make sure to follow the [conventional commit specification](https://conventionalcommits.org/) before contributing. To help you with commit a commit template is provide. 
-Run `config.sh` to initialize the commit template to your `.git/config`. 
+Make sure to follow the [conventional commit specification](https://conventionalcommits.org/) before contributing. To help you with commit a commit template is provide. Run `config.sh` to initialize the commit template to your `.git/config` or use [commitizen](https://www.npmjs.com/package/commitizen)
 
 Once you are finished with your changes, run `npm test` to make sure all tests pass.
 
-Do a pull request against `development`, make sure the build passes. A team member will review and merge your pull request. 
-Once merged from `development` to `master` one pull request will be created against `development`. Merge the pull request then create another pull request from `development` into `master`. Once the final pull request has been merged the generator will be published to npm.  Make sure that the CHANGELOG.md and the package.json is correct before merging the auto generated pull request. 
+Do a pull request against `master`, make sure the build passes. A team member will review and merge your pull request.
+Once merged to `master` an auto generated  pull request will be created against `master`. Make sure that the CHANGELOG.md and the package.json is correct before merging the pull request. After the auto generated pull request has been merged to `master` the version will be bumped and published to npm.
```

Comment:
Suggested change for clarity:
Once merged to `master` an auto generated  pull request will be created against `master`.
changed to:
Once merged to `master` an auto generated  pull request will be created against `master` to update the changelog.